### ISSUE TITLE
Invalid Icon annotation in Modelica.Blocks.MathInteger.MultiSwitch.

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1302]
**Reported by dhedberg on 2 Oct 2013 15:16 UTC**
Modelica.Blocks.MathInteger.MultiSwitch has an invalid Icon annotation. The textAttribute of a Text primitive is missing the DynamicSelect expression:

```
textString = String(y, minimumLength = 1, significantDigits = 0)
```

Should probably be:

```
textString = DynamicSelect(" ", String(y, minimumLength = 1, significantDigits = 0))
```

This error is also present on trunk.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1302

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 7 Oct 2013 08:15 UTC**

**Comment by modelica-trac-importer** [comment]
**Modified by beutlich on 16 May 2014 15:07 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by beutlich on 16 May 2014 20:46 UTC**
Resolved by 8eb73e099cf6e0e92a941f1320295f2eebdb107b and 4b5637ad85d39c0307e47c296027664897d1d974.
